pilot/lib/pilot/tester_manager.rb

```diff
@@ -126,8 +127,20 @@ def find_app_tester(email: nil, app: nil)
       return tester
     end
 
+    def find_current_user
+      current_user_email = Spaceship::Tunes.client.user_email
+      current_user_apple_id = Spaceship::Tunes.client.user
+
+      current_user = Spaceship::Members.find(current_user_email)
+      unless current_user
+        UI.user_error!("Unable to find a member for AppleID: #{current_user_apple_id}, email: #{current_user_email}")
+        return nil
```

Comment:
I agree, my thinking is: "What if we inject a different implementation that doesn't actually cause us to bail?" in that case, I'd rather signal my intention and make sure nothing else runs... just in case. What do you think? Is that overkill?